### DISCUSSION TITLE
Is there an online chat to ask some questions about insllation and usage

### DISCUSSION THREAD

**Discussion by Movix** [eldy/SellYourSaas#357] [General]
Hello,
I'm currently trying to get sellyoursass installed and running.

So far i got the administration interface (dolibarr + sellyoursaas plugin) running.
I struggle with setting things correctly up within the management interface.
I believe that i made some mistake or error during the installation.
So far i set up a package and various services and i'm able to register as a new customer on the myaccount page.
The main issues i'm running into are that i do not understand how to setup the database dump correctly so that the created customer database gets populated. The database is created but empty.
Also the myaccount page seems to work well (except the errors related to the missing dolibarr database tables) but as soon as i disconnect i'm unable to reconnect. User / password combination are refused claiming they're wrong but  i'm sure they are correct.

Is there any place where i can ask questions like this without disturbing those working on the developement of the project ?

Movix

**Comment by priojk** [comment]
Hi,

regarding the database dump, it's important that it needs to be unzipped (plain SQL). It goes into the package under the tab "Linked files" - you just upload it there and it will then be used to fill a newly created database.

Overall, I can confirm that the entire solution does indeed work, but setting it up is a bit tedious and you need to follow along the entire documentation precisely as stated.

**Comment by Movix** [comment]
@priojk Thank you for your reply. At least i got this SQL dump thing sorted out thanks to your answer.
